### PR TITLE
Approved App Listing (updated)

### PR DESCRIPTION
**Pull request by silverbucket**
I've taken @AngeloR's PR (https://github.com/remotestorage/armadietto/pull/47) and updated it with the improved styling, resolved some conflicts, but otherwise left it as-is.

The PR looks good to me, and the only comment from @raucao was that we add a revoke button as part of the app listing. A quick look through the code and it looks like there is no API functionality to do this yet, so I think it's out of scope of this PR. We can create an issue to implement that. 

Aside from any other issues I overlooked, I propose we go ahead and merged this.

### CODE REVIEW

File: lib/controllers/users.js
Comment (DougReeder) [inline lib/controllers/users.js:85]: Can we verify that a caught error will result in a reasonable message to the user, and a useful message in the log?

**Review by DougReeder — CHANGES_REQUESTED** [review]
For my dev server, the app listing looks like this:
Screenshot 2024-01-01 at 6 47 46 PM

1. Inspektor appears twice. Is that a bug in this PR, or somewhere else?
2. All of the favicons are blocked by our Content Security Policy. To allow them, we'd have to allow images from **any** site.  I don't think the value is worth the risk.
3. Not all apps store their favicon at this location, so we'd miss some available favicons

I think we should drop the favicons entirely.

**Comment by DougReeder** [comment]
This list just shows what's in `auth.json`, so the duplicate of inspektor is not a bug in this PR.

**Comment by DougReeder** [comment]
The error message is as good as we can do, until #106 is merged.

File: lib/controllers/users.js
Comment (DougReeder) [reply lib/controllers/users.js:85]: The error message is as good as we can do, until https://github.com/remotestorage/armadietto/pull/106 is merged.

**Comment by silverbucket** [comment]
> I think we should drop the favicons entirely.

@DougReeder  Agreed, I've removed them. Tests are passing, so with your approval I'll merge.

**Review by DougReeder — APPROVED** [review]

**Comment by DougReeder** [comment]
We're good to merge!

**Pull request merged**
